gradle/versions.gradle

```diff
@@ -17,7 +20,20 @@ ext.libVersions = [
     hamcrest: '1.3',
     commons_codec: '1.6',
     libthrift: '0.9.2',
-    protobuf: '2.6.0'
+    protobuf: '2.6.0',
+    findbugs: '1.3.9',
+    hikariCP: '2.4.5',
+
+    // Danger, Will Robinson!
+    //
+    // This driver has historically low code quality and should be considered a high-risk update.
+    // Severe performance regressions in 1202,3,4.
+    // Severe correctness issues in 1204,5,6.
+    // Update with care and caution.
+    postgresql: '9.4.1207.jre7',
+
+    c3p0: '0.9.5.1',
+    log4j: '1.2.17'
```

Comment:
not sure how this is going to play with logback, I assume folks will have to explicitly exclude log4j?